src/main/java/io/tiledb/java/api/Array.java

```diff
@@ -525,6 +525,11 @@ public static void consolidate(
     consolidate(ctx, uri, encryption_type, key, new Config());
   }
 
+  public static void consolidate() {
+    // TODO consolidate fragments. This method seems unstable in the core library for now. Will be
```

Comment:
What is the problem here? Is there a shortcut story about this?

Reply:
According to the core library:

_Note: This API needs to be used with caution until we implement consolidation with timestamps. For now, if the non-empty domain of the consolidated fragments overlap anything in the fragments that come in between, this could lead to unpredictable behavior._

Since TileDB-Java does not have an option to disable experimental features, I decided to leave it until a more stable version is out.